screensaver.digitalclock/resources/skins/default/720p/skin.carpc-carbon.xml

```diff
@@ -0,0 +1,239 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
it's a good practice to keep the xml files with the name of the script so that it doesn't get confused with the skin files
something like screensaver-digitalclock-carpc-carbon.xml would work for you?

Reply:
That wouldn't work since in the code I pull what the skin is and then match that with the xml files.
I have to maintain these files because Kodi won't allow addons to define their own fonts - which frustrated me more than it should and made the development of this addon far more complicated and time consuming than it should be.
In order for your suggestion to work I'd have to change the main file (which would require some research as I'm not a programmer) and then proper testing.
I'd like to get this fix out first and then later on decide if I'll continue to work on this screensaver any longer.